docs/Agent/user-guide/guides/end-to-end-workflow.md

```diff
@@ -0,0 +1,21 @@
+### End-to-End Workflow
+
+Each command in pbench-agent accepts the `--help` option and outputs a brief usage message
+
+The default set of tools for data collection can be enabled with
+
+	$pbench-register-tool-set
+
+To list all your registered tools
+
+	$pbench-list-tools
+
+You may then perform a built-in benchmark by running it's Pbench script
+
+	$pbench-user-benchmark – sleep 10
+
+The above command will collect data from the registered tools for the specified time period and save it in the `/var/lib/pbench-agent` directory.
```

Comment:
While not completely inaccurate, the "save it *in*" bothers me as the agent will create a bunch of distinct directories to separate the datasets. In fact, this might be a good time to (very briefly) introduce the naming convention for performance runs, with the benchmark name, config string, and the run timestamp.